sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -95,6 +112,7 @@ def create_key(self, name, key_type, **kwargs):
             tags=kwargs.pop("tags", None),
             curve=kwargs.pop("curve", None),
             public_exponent=kwargs.pop("public_exponent", None),
+            release_policy=policy,
```

Comment:
Is there test coverage of `policy != None`?

Reply:
Do you mean testing `create_key` with a non-`None` release policy?

Reply:
Yes, I didn't see/perhaps overlooked a test of creating a key with a release policy

Reply:
Got it! That's covered in [this test](https://github.com/Azure/azure-sdk-for-python/pull/20048/files#diff-20c2e1ce15e9fdb884cf57de69b095ef3e6cb6683e74d84668f18c25789a496bR450), though it's a little obscured by the helper method